x/assets/keeper/staker_asset.go

```diff
@@ -31,6 +33,38 @@ func (k Keeper) GetStakerAssetInfos(ctx sdk.Context, stakerID string) (assetsInf
 }
 
 func (k Keeper) GetStakerSpecifiedAssetInfo(ctx sdk.Context, stakerID string, assetID string) (info *assetstype.StakerAssetInfo, err error) {
+	if assetID == assetstype.NativeAssetID {
+		stakerAddrStr, _, err := assetstype.ParseID(stakerID)
+		if err != nil {
+			return nil, err
+		}
+		stakerAcc := sdk.AccAddress(hexutil.MustDecode(stakerAddrStr))
```

Comment:
Since this function can be called via `grpc`, don't use `MustDecode` directly.

Reply:
updated